flowforge/input/BoundaryConditions.py

```diff
@@ -132,6 +132,42 @@ def _convertUnits(self, uc: UnitConverter) -> None:
             else:
                 raise Exception("Unknown enthalpy BC type: " + self.type_outlet)
 
+class VoidBC:
+    """Class for void fraction BC
+
+    Attributes
+    ----------
+    mdot : float
+        Mass float rate [kg/s]
```

Comment:
Would we want to support `mdot` and `void_fraction`?  Maybe not?  Just wanted to think about it a little.

Reply:
Same as above